src/test/features/offer/form/models/offer.ts

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'chai'
+import { Validator } from 'class-validator'
+import { expectValidationError } from '../../../../app/forms/models/validationUtils'
+import { Offer } from 'offer/form/models/offer'
+import { ValidationErrors } from 'offer/form/models/offer'
+import { LocalDate } from 'forms/models/localDate'
+import * as moment from 'moment'
+
+describe('Offer', () => {
+  describe('constructor', () => {
+    it('should set the fields to undefined', () => {
+      const offer = new Offer()
+      expect(offer.offerText).to.be.equal(undefined)
+      expect(offer.completionDate).to.be.equal(undefined)
+    })
+  })
+
+  describe('form object deserialization', () => {
+    it('should return undefined when value is undefined', () => {
+      expect(Offer.fromObject(undefined)).to.be.equal(undefined)
+    })
+
+    it('should return null when value is null', () => {
```

Comment:
We are trying to avoid programming against `null` value in JavaScript and treat `undefined` as the go-to missing/blank value. This test case can be removed.

Reply:
we don't use null in the code, but this is just test which tests for null, extra safety check should be all right ??

Reply:
no we don't use null, so we shouldn't test against it